ci/docker-rust/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN set -x \
  && cargo install mdbook \
  && cargo install mdbook-linkcheck \
  && cargo install svgbob_cli \
- && cargo install --git https://github.com/rustwasm/wasm-pack --rev b4e619c8a13a8441b804895348afbfd4fb1a68a3 \
+ && cargo install wasm-pack \
  && cargo install sccache \
```

Comment:
(yet another) off-topic: https://github.com/solana-labs/solana/blob/cc6c4540766d595a3720e39346b4500f2e2d982c/ci/test-stable.sh#L49 yet another `cargo install` candidate to be pulled in here.

Reply:
I found we don't run stable-sbf in the docker. maybe we need to make it run in the docker then add it to the list 🤔